mteb/tasks/BitextMining/multilingual/BibleNLPBitextMining.py

```diff
@@ -0,0 +1,955 @@
+    "vie_Latn",  # Vietnamese
+    "viv_Latn",  # Iduna
+    "vmy_Latn",  # Mixtec, many Mixtec languages use Latin script, though there are multiple variants of Mixtec.
+    "waj_Latn",  # Waffa
+    "wal_Ethi",  # Wolaytta, uses the Ethiopic script
+    "wap_Latn",  # Wapishana
+    "wat_Latn",  # Kaninuwa
+    "wbi_Latn",  # Vwanji
+    "wbp_Latn",  # Warlpiri
+    "wed_Latn",  # Wedau
+    "wer_Latn",  # Weri
+    "wim_Latn",  # Wik-Mungkan
+    "wiu_Latn",  # Wiru
+    "wiv_Latn",  # Vitu
+    "wmt_Latn",  # Mwani
+    "wmw_Latn",  # Mwani
+    "wnc_Latn",  # Wantoat
+    "wnu_Latn",  # Usan
+    "wol_Latn",  # Wolof, primarily uses Latin script but also uses Arabic in some contexts
+    "wos_Latn",  # Hanga Hundi
+    "wrk_Latn",  # Garrwa
+    "wro_Latn",  # Worrorra
+    "wrs_Latn",  # Waris
+    "wsk_Latn",  # Waskia
+    "wuv_Latn",  # Wuvulu-Aua
+    "xav_Latn",  # Xavante
+    "xbi_Latn",  # Kombio
+    "xed_Latn",  # Hdi
+    "xla_Latn",  # Kamula
+    "xnn_Latn",  # Northern Kankanay
+    "xon_Latn",  # Konkomba
+    "xsi_Latn",  # Sio
+    "xtd_Latn",  # Diuxi-Tilantongo Mixtec
+    "xtm_Latn",  # Magdalena Peñasco Mixtec
+    "yaa_Latn",  # Yaminahua
+    "yad_Latn",  # Yagua
+    "yal_Latn",  # Yalunka
+    "yap_Latn",  # Yapese
+    "yaq_Latn",  # Yaqui
+    "yby_Latn",  # Yawalapití
+    "ycn_Latn",  # Yucuna
+    "yka_Latn",  # Yakan
+    "yle_Latn",  # Yele
+    "yml_Latn",  # Iamalele
+    "yon_Latn",  # Yongkom
+    "yor_Latn",  # Yoruba, primarily uses the Latin script
+    "yrb_Latn",  # Yareba
+    "yre_Latn",  # Yaouré
+    "yss_Latn",  # Yessan-Mayo
+    "yuj_Latn",  # Karkar-Yuri
+    "yut_Latn",  # Yopno
+    "yuw_Latn",  # Yau (Morobe Province)
+    "yva_Latn",  # Yawa
+    "zaa_Latn",  # Sierra de Juárez Zapotec
+    "zab_Latn",  # San Juan Guelavía Zapotec
+    "zac_Latn",  # Ocotlán Zapotec
+    "zad_Latn",  # Cajonos Zapotec
+    "zai_Latn",  # Isthmus Zapotec
+    "zaj_Latn",  # Zaramo
+    "zam_Latn",  # Miahuatlán Zapotec
+    "zao_Latn",  # Ozolotepec Zapotec
+    "zap_Latn",  # Zapotec, uses Latin script across many varieties
+    "zar_Latn",  # Rincón Zapotec
+    "zas_Latn",  # Santo Domingo Albarradas Zapotec
+    "zat_Latn",  # Tabaa Zapotec
+    "zav_Latn",  # Yatzachi Zapotec
+    "zaw_Latn",  # Mitla Zapotec
+    "zca_Latn",  # Coatecas Altas Zapotec
+    "zga_Latn",  # Ganza
+    "zia_Latn",  # Zia
+    "ziw_Latn",  # Zigula
+    "zlm_Latn",  # Malay (individual language), Latin script
+    "zos_Latn",  # Francisco León Zoque
+    "zpc_Latn",  # Choapan Zapotec
+    "zpl_Latn",  # Lachixío Zapotec
+    "zpm_Latn",  # Mixtepec Zapotec
+    "zpo_Latn",  # Amatlán Zapotec
+    "zpq_Latn",  # Zoogocho Zapotec
+    "zpu_Latn",  # Yalálag Zapotec
+    "zpv_Latn",  # Chichicapan Zapotec
+    "zpz_Latn",  # Texmelucan Zapotec
+    "zsr_Latn",  # Southern Rincon Zapotec
+    "ztq_Latn",  # Quioquitani-Quierí Zapotec
+    "zty_Latn",  # Yatee Zapotec
+    "zyp_Latn",  # Zyphe Chin
+]
+
+_SPLIT = ["train"]
```

Comment:
Probably good to note why the train split is chosen here since there are also validation and test splits a well.

Reply:
Makes sense. I went for the train split because the validation/test splits are extremely small in a lot of cases (think 20 sentences). Given this I thought it would be better to subsample the train set.